tests/Capgemini.PowerApps.PackageDeployerTemplate.IntegrationTests/PackageTemplateBaseTests.cs

```diff
@@ -136,5 +137,20 @@ public void PackageTemplateBase_FlowsInPackage_AreActiveAccordingToConfig(string
 
             workflow["statecode"].As<OptionSetValue>().Value.Should().Be(stateCode);
         }
+
+        [Theory]
+        [InlineData("account", "pdt_testautonumber", 1)]
```

Comment:
Could we add the contact filed as well here?

Reply:
I hadn't added the contact actually. I refer to it in the XML as an example. Part of me wanted to see if it would fall over if it had something that doesn't exist (it doesn't). I can add the contact entity if we want it, might be valuable to make sure we can do multiple :)

Reply:
No worries then, as long as it was intentional.